### PR TITLE
HtmlSheetSupport: convert to SimpleHtmlPanel

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/pcgen/gui2/InfoGuidePane.java [hunk 1]
```diff
@@ -122,16 +122,16 @@ private void initComponents()
...
-			boolean show = e.getNewReference() == null;
-			if (show)
-			{
-				tipPane.getController().setHtml(
-						LanguageBundle.getFormattedString(
-								"in_si_tip",
-								TipOfTheDayHandler.getInstance().getNextTip()
-						));
-			}
+			frame.getSelectedCharacterRef().addReferenceListener(e -> {
```
Comment (thpr) [inline code/src/java/pcgen/gui2/InfoGuidePane.java]: Why the indentation change?

File: code/src/java/pcgen/gui2/tabs/models/HtmlSheetSupport.java [hunk 2]
```diff
@@ -118,7 +105,7 @@ public void refresh()
 		}
 		if (refresher != null && !refresher.isDone())
 		{
-			refresher.cancel(false);
+			refresher.cancel(true);
```
Comment (thpr) [inline code/src/java/pcgen/gui2/tabs/models/HtmlSheetSupport.java]: Is this an embedded bug fix, or ?

Comment (grimreaper) [reply code/src/java/pcgen/gui2/tabs/models/HtmlSheetSupport.java]: yes. this is a bug fix. or at least a behavior change that I think is better. Will make it a separate PR for better future debugging.

File: code/src/java/pcgen/gui2/InfoGuidePane.java (hunk 1 above)
Comment (grimreaper) [reply code/src/java/pcgen/gui2/InfoGuidePane.java]: bug. fixed.

**Comment by grimreaper** [comment]
This needs to be recreated.

**Pull request closed without merging**